.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,22 +1,22 @@
-<!--
-Thanks for contributing to Accentor!
-Make sure all GitHub actions (test & lint) will pass and fill out the template.
+<!---
+  Thanks for contributing to Accentor!
+  Make sure all GitHub actions (test & lint) will pass and fill out
+  the template.
 
-You can tag your PR with the relevant tags and request a review from someone of the team.
```

Comment:
This was not actually true, only collaborators can edit tags and request reviews.